src/Uno.UI/UI/Xaml/Controls/Repeater/RecyclePool.cs

```diff
@@ -36,6 +36,11 @@ public UIElement TryGetElement(string key)
 		public UIElement TryGetElement(string key, UIElement owner)
 			=> TryGetElementCore(key, owner);
 
+		internal void Clear()
+		{
+			m_elements.Clear();
+		}
```

Comment:
Please flag uno specific code in files imported from WinUI, eg:

```suggestion
#if HAS_UNO
		// Uno specific: Helper used by template updater
		internal void Clear()
		{
			m_elements.Clear();
		}
#endif
```